src/app/features/notifications/notifications.component.ts

```diff
@@ -0,0 +1,134 @@
+import { Component, OnInit } from "@angular/core";
+import { MatBadgeModule } from "@angular/material/badge";
+import { NgIf } from "@angular/common";
+import { FontAwesomeModule } from "@fortawesome/angular-fontawesome";
+import { MatMenu } from "@angular/material/menu";
+import { NgFor } from "@angular/common";
+import { MatButtonModule } from "@angular/material/button";
+import { MatMenuTrigger } from "@angular/material/menu";
+import { MatMenuModule } from "@angular/material/menu";
+import { FormsModule } from "@angular/forms";
+import { MatTooltipModule } from "@angular/material/tooltip";
+import { Logging } from "app/core/logging/logging.service";
+import { NotificationEvent } from "./model/notification-event";
+import { EntityMapperService } from "app/core/entity/entity-mapper/entity-mapper.service";
+import { MatTabsModule } from "@angular/material/tabs";
+import { NotificationItemComponent } from "./notification-item/notification-item.component";
+import { MockNotificationsService } from "./mock-notifications.service";
+import { CurrentUserSubject } from "app/core/session/current-user-subject";
+
+@Component({
+  selector: "app-notifications",
+  standalone: true,
+  imports: [
+    MatBadgeModule,
+    NgIf,
+    FontAwesomeModule,
+    MatMenu,
+    NgFor,
+    MatButtonModule,
+    MatMenuTrigger,
+    MatMenuModule,
+    FormsModule,
+    MatTooltipModule,
+    MatTabsModule,
+    NotificationItemComponent,
+  ],
+  templateUrl: "./notifications.component.html",
+  styleUrl: "./notifications.component.scss",
+})
+export class NotificationsComponent implements OnInit {
+  public allNotifications: NotificationEvent[] = [];
+  public unreadNotifications: NotificationEvent[] = [];
+  public hasUnreadNotificationCount = 0;
+  public isEnableNotification = false;
+  public selectedTab = 0;
+
+  constructor(
+    private entityMapper: EntityMapperService,
+    private mockNotificationsService: MockNotificationsService,
+    private currentUser: CurrentUserSubject,
```

Comment:
--> use SessionInfo as discussed?

Reply:
Yes. Now, I have used the `SessionInfo`.